docs/reference/cluster-config.txt

```diff
@@ -288,6 +288,32 @@ The master supports the following configuration settings:
     to all GPU tasks when running on Kubernetes. See :ref:`custom-pod-specs`
     for details.
 
+  - ``image``: Defines the default docker image to use when executing the workload.
+    If a docker image is specified in the :ref:`experiment config <exp-environment-image>`
+    this default is overriden. This image must be accessible via ``docker pull`` to every
+    Determined agent machine in the cluster. Users can use different container images for GPU
+    vs. CPU agents differently by specifying a dict with two keys, ``cpu`` and ``gpu``.
+    Default values:
+
+    - ``determinedai/environments:py-3.6.9-pytorch-1.4-tf-1.15-cpu-0.5.0`` for CPU agents
+    - ``determinedai/environments:cuda-10.0-pytorch-1.4-tf-1.15-gpu-0.5.0`` for GPU agents.
+
+  - ``force_pull_image``: Defines the default policy for forcibly pulling images from
+    the docker registry and bypassing the docker cache. If a pull policy is specified
+    in the :ref:`experiment config <exp-environment-image>` this default value is
+    overriden. Defaults to ``false``.
```

Comment:
As of November 1st, unauthenticated users will be capped at 100 pulls per 6 hours. This limit is based on IP address. We might want to call out that `force_pull_image=True` could lead to that cap being hit very quickly.

https://www.docker.com/blog/scaling-docker-to-serve-millions-more-developers-network-egress/